### ISSUE TITLE
Customize GitHub language detection

### ISSUE THREAD

**Issue by arey** [spring-petclinic/spring-petclinic-reactjs#5]
GitHub uses [linguist](https://github.com/github/linguist) to detect languages. For spring-petclinic-reactks, CSS is the first one. It should be Java or TypeScript.
We could customize this behavior, See https://github.com/spring-petclinic/spring-petclinic-kotlin/blob/master/.gitattributes

**Comment by ankurshukla10** [comment]
@arey  I can send in a pull request for this change, seems pretty straightforward. Anything specific I should take care of? Thanks !!

**Comment by arey** [comment]
Hi @ankurshukla10 , sure, you could submit a change. If you want to upgrade this sample to more recent versions of Spring Boot and ReactJS, you're welcome.

![image](https://user-images.githubusercontent.com/838318/101650223-03bb7e00-3a3c-11eb-9461-c7c1e17266b5.png)

**Comment by arey** [comment]
After your PR merge, the languages repartition didn't changed and I don't know why :-(

**Comment by ankurshukla10** [comment]
@arey  yeah, I noticed too :(. I am trying to check this locally by installing github-linguist on my development machine, but to my bad luck the install keeps on failing. Right now I am getting the attached error. Do you have any insights on why that might be? Or If you can point me in a better direction, that'll be helpful.

_Machine-Processor:_ Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, 2112 Mhz, 4 Core(s), 8 Logical Processor(s)
_OS Name:_ Microsoft Windows 10 Pro (Version-10.0.19041 Build 19041)
_Ruby Version:_ ruby 2.6.5p114 (2019-10-01 revision 67812) [x64-mingw32]
_Gem Version:_ 3.0.3



[github-linguist-install-fails.txt](https://github.com/spring-petclinic/spring-petclinic-reactjs/files/5671128/github-linguist-install-fails.txt)

**Comment by arey** [comment]
Maybe you have to install a ruby gem?
```
gem install ffi -- --enable-system-libffi        # to install the gem manually
bundle config build.ffi --enable-system-libffi   # for bundle install
```
See https://github.com/ffi/ffi
